sql/updates/world/master/XXXX_XX_XX_XX_world.sql

```diff
@@ -0,0 +1,259 @@
+(269054, 'spell_boralus_find_cyrus_objective_complete');
+
+-- Scene
+DELETE FROM `scene_template` WHERE `SceneId`=1960;
+INSERT INTO `scene_template` (`SceneId`, `Flags`, `ScriptPackageID`, `Encrypted`) VALUES
+(1960, 17, 2107, 0);
+
+UPDATE `scene_template` SET `ScriptName` = 'scene_boralus_client_scene_cyrus_and_genn' WHERE `SceneId` = 1960;
+
+-- Gossip
+DELETE FROM `creature_template_gossip` WHERE (`CreatureID`=122370 AND `MenuID`=22543) OR (`CreatureID`=120788 AND `MenuID`=23317);
+INSERT INTO `creature_template_gossip` (`CreatureID`, `MenuID`, `VerifiedBuild`) VALUES
+(122370, 22543, 52808), -- Cyrus Crestfall
+(120788, 23317, 53040); -- Genn Greymane
+
+UPDATE `creature_template_gossip` SET `VerifiedBuild`=52808 WHERE (`CreatureID`=121235 AND `MenuID`=21307);
+
+DELETE FROM `gossip_menu` WHERE (`MenuID`=21307 AND `TextID`=133576);
+INSERT INTO `gossip_menu` (`MenuID`, `TextID`, `VerifiedBuild`) VALUES
+(21307, 133576, 52808); -- 121235 (Taelia)
+
+UPDATE `gossip_menu_option` SET `GossipOptionID`=48242, `VerifiedBuild`=53040 WHERE (`MenuID`=22543 AND `OptionID`=0);
+
+-- Quest stuff
+DELETE FROM `creature_questender` WHERE (`id`=121235 AND `quest`=46729);
+INSERT INTO `creature_questender` (`id`, `quest`, `VerifiedBuild`) VALUES
+(121235, 46729, 52808); -- The Old Knight ended by Taelia
+
+-- Conversation
+DELETE FROM `conversation_template` WHERE `Id` IN (7653, 8062, 7605);
+INSERT INTO `conversation_template` (`Id`, `FirstLineID`, `TextureKitId`, `VerifiedBuild`) VALUES
+(7653, 17449, 0, 52808),
+(8062, 18297, 0, 52808),
+(7605, 17297, 0, 52808);
+
+UPDATE `conversation_template` SET `ScriptName` = 'conversation_boralus_accept_old_knight' WHERE `Id` = 9556;
+UPDATE `conversation_template` SET `ScriptName` = 'conversation_boralus_enter_harbormaster_office' WHERE `Id` = 7605;
+UPDATE `conversation_template` SET `ScriptName` = 'conversation_boralus_cyrus_meets_genn' WHERE `Id` = 8062;
+UPDATE `conversation_template` SET `ScriptName` = 'conversation_cyrus_crestfall_shaking_hands' WHERE `Id` = 7653;
+
+DELETE FROM `conversation_actors` WHERE (`Idx`=1 AND `ConversationId` IN (8062,7653)) OR (`Idx`=0 AND `ConversationId` IN (8062,7605,7653));
+INSERT INTO `conversation_actors` (`ConversationId`, `ConversationActorId`, `ConversationActorGuid`, `Idx`, `CreatureId`, `CreatureDisplayInfoId`, `NoActorObject`, `ActivePlayerObject`, `VerifiedBuild`) VALUES
+(8062, 0,(@CGUID+3), 1, 0, 0, 0, 0, 52808), -- Full: 0x20309CCD607664C00000EA00001E0614 Creature/0 R3111/S234 Map: 1643 (Kul Tiras) Entry: 121235 (Taelia) Low: 1967636
+(8062, 59635, 0, 0, 0, 0, 0, 0, 52808), -- Full: 0x20309CCD607780800000EA00001E0614 Creature/0 R3111/S234 Map: 1643 (Kul Tiras) Entry: 122370 (Cyrus Crestfall) Low: 1967636
+(7605, 59582, 0, 0, 0, 0, 0, 0, 52808), -- Full: 0x20309CCD6079B5800000EA000021CFE2 Creature/0 R3111/S234 Map: 1643 (Kul Tiras) Entry: 124630 (Kul Tiran Guard) Low: 2215906
+(7653, 64106, (@CGUID+0), 0, 0, 0, 0, 0, 52808), -- Full: 0x0
```

Comment:
i assume conversation is started without Genn and then he gets added as this actor at some point?

Reply:
Nope, just Cyrus get cloned. Genn is added as actor in sql